docs/src/__examples__/CountryFlag/DEFAULT.tsx

```diff
@@ -3,6 +3,11 @@ import { CountryFlag } from "@kiwicom/orbit-components";
 
 export default {
   Example: () => <CountryFlag />,
+  info: {
```

Comment:
I removed all `info` in examples, since we do not need them anymore, could you remove?